GOVERNANCE.md

```diff
@@ -0,0 +1,41 @@
+# Governance
+
+## Project Maintainers
+
+[Project maintainers](MAINTAINERS.md) are responsible for activities around
+maintaining and updating the KEDA. Final decisions on the features
+reside with the project maintainers.
+
+Maintainers MUST remain active. If they are unresponsive for >3 months, they
+will be automatically removed unless a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
+the other project maintainers agrees to extend the period to be greater than 3
+months.
+
+New maintainers can be added to the project by a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
+vote of the existing maintainers. A potential maintainer may be nominated by an
+existing maintainer. A vote is conducted in private between the current
+maintainers over the course of a one week voting period. At the end of the week,
+votes are counted and a pull request is made on the repo adding the new
+maintainer to the [MAINTAINERS](MAINTAINERS.md) file.
+
+A maintainer may step down by submitting an
+[issue](https://github.com/kedacore/keda/issues/new) stating their intent.
+
+Changes to this governance document require a pull request with approval from a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
+the current maintainers.
+
+## Managing Projects
+
+New projects can be added to the organization after a successful
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
+vote of the existing maintainers.
+
+Contributors who want to donate a scaler, sample or other contribution must
```

Comment:
I suppose you were about to change this part in the previous PR

Reply:
Good point, forgot it